### ISSUE TITLE
Mask as a command line argument?

### ISSUE THREAD

**Issue by StanislavZayarsky** [Sanster/lama-cleaner-docs#54]
Thank you so much for such a great tool! :)

I wonder if is there an easy way to pass mask image via the command line so we automate processing without launching GUI?

**Comment by Sanster** [comment]
Yes, you can use the `iopaint run` command in the command line (lama-cleaner has been renamed to iopaint): https://www.iopaint.com/batch_process
